package account.exception

class UserAlreadyExistsException(message: String) : RuntimeException(message)
